developer-docs-site/docs/tutorials/first-coin.md

```diff
@@ -131,8 +136,19 @@ Bob's initial MoonCoin balance: 0.
 Alice mints Bob some of the new coin.
 Bob's updated MoonCoin balance: 100.
 ```
+
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+    Coming soon.
+
+  </TabItem>
+</Tabs>
```

Comment:
Uh, this doesn't belong down here?  Should go back above step 3.1

Reply:
This is a bit of a mess... we insist on giving examples with all 3 SDKs - ts, python and rust, but each has different implementation and rust doesnt have any implementation.

It would have been easier to see if there was a Netlify preview, but step 3.1 and 3.2 should be under the python implementation.